pkgs/top-level/aliases.nix

```diff
@@ -92,6 +92,7 @@ mapAliases ({
   bird2 = bird; # Added 2022-02-21
   bitwig-studio1 = throw "bitwig-studio1 has been removed, you can upgrade to 'bitwig-studio'"; # Added 2023-01-03
   bitwig-studio2 = throw "bitwig-studio2 has been removed, you can upgrade to 'bitwig-studio'"; # Added 2023-01-03
+  blender-with-packages = args: lib.warn "blender-with-packages is deprecated and will be removed in later versions" blender.withPackages (_: args.packages) { name = args.name; }; # Added 2023-10-30
```

Comment:
Consider advertising the alternative here too: withPackages. Then it's clearer that there's a new method instead of it sounding like it's not supported at all anymore.

Reply:
Changed message to be more clearer, thought that line has become quite a bit longer...

Reply:
Don't worry about it. Also, the message should probably refer people to `blender.withPackages`.

Reply:
> Don't worry about it. Also, the message should probably refer people to `blender.withPackages`.

The selected line was outdated. 

Current version is -

`blender-with-packages is deprecated in favor of blender.withPackages, and will be removed in future releases`

Reply:
Maybe something like `blender-with-packages is deprecated, please use blender.withPackages(ps: [ ps.example ])`? Something that has an example, so it's clearer for how to migrate.

Reply:
> Maybe something like `blender-with-packages is deprecated, please use blender.withPackages(ps: [ ps.example ])`? Something that has an example, so it's clearer for how to migrate.

I would think that people would look at the changelog for it... hmm.

I want to add "Similar to python3.withPackages" to the end of it too, would that be too long?

Reply:
I just think it's should be `blender-with-packages is deprecated, please use blender.withPackages, which behaves similar to python3.withPackages.`, probably removing the last part for brevity